lib/view/fiscaat/BeheerCiviSaldoView.class.php

```diff
@@ -46,4 +47,19 @@ public function __construct(CiviSaldo $model) {
 	}
 }
 
-class BeheerSaldoResponse extends DataTableResponse {}
\ No newline at end of file
+class BeheerSaldoResponse extends DataTableResponse {
+	/**
+	 * @param CiviSaldo $entity
+	 * @return string
+	 */
+	public function getJson($entity) {
+		$data = array(
+			'uid' => $entity->uid,
+			'naam' => ProfielModel::getNaam($entity->uid, 'volledig'),
+			'lichting' => substr($entity->uid, 0, 2),
+			'saldo' => $entity->saldo,
+			'laatst_veranderd' => $entity->laatst_veranderd
+		);
+		return json_encode($data);
```

Comment:
Dit is iets netter: https://github.com/csrdelft/csrdelft.nl/blob/master/lib/view/formulier/DataTable.class.php#L474